### PR TITLE
Bug 1997509: delete only necessary resources instead of all

### PR DESCRIPTION
**Pull request by atiratree**

### CODE REVIEW

File: test/extended/builds/pipeline_origin_bld.go [hunk 1]
```diff
@@ -255,14 +255,6 @@ var _ = g.Describe("[sig-builds][Feature:Builds][sig-devex][Feature:Jenkins][Slo
 					g.By("get build console logs and see if succeeded")
 					_, err = j.GetJobConsoleLogsAndMatchViaBuildResult(brservices, "Finished: SUCCESS")
 					o.Expect(err).NotTo(o.HaveOccurred())
-					g.By("clean up openshift resources for next potential run")
```
Comment (soltysh) [inline test/extended/builds/pipeline_origin_bld.go:258]: Why removing these?

**Review by soltysh — COMMENTED** [review]
One question, otherwise this looks good
/approve

File: test/extended/builds/pipeline_origin_bld.go (hunk 1 above)
Comment (atiratree) [reply test/extended/builds/pipeline_origin_bld.go:258]: @soltysh The removal of e2e namespace should cleanup these by default. I removed them for better readability/less clutter but I can revert the change if you prefer.

Comment (soltysh) [reply test/extended/builds/pipeline_origin_bld.go:258]: You're right, I just noticed `defer` at the beginning of the test.

**Review by soltysh — APPROVED** [review]
/lgtm

**Pull request merged**